applications/nrf5340_audio/README.rst

```diff
@@ -505,14 +505,14 @@ Selecting the BIS mode
 ======================
 
 The CIS mode is the default operating mode for the application.
-You can switch to the BIS mode by adding the :kconfig:option:`CONFIG_TRANSPORT_BIS` Kconfig option set to ``y`` to the :file:`prj.conf` file for the debug version and the :file:`prj_release.conf` file for the release version.
+To switch to the BIS mode, add and set the :ref:`CONFIG_TRANSPORT_BIS <config_nrf53_audio_app_options>` Kconfig option to ``y`` to the :file:`prj.conf` file for the debug version and the :file:`prj_release.conf` file for the release version.
```

Comment:
`To switch to the BIS mode, add and set the :ref:`CONFIG_TRANSPORT_BIS <config_nrf53_audio_app_options>` Kconfig option to ``y`` to the :file:`prj.conf` file for the debug version and the :file:`prj_release.conf` file for the release version.`
->
`To switch to the BIS mode, set the :ref:`CONFIG_TRANSPORT_BIS <config_nrf53_audio_app_options>` Kconfig option to ``y`` in the :file:`prj.conf` file for the debug version and the :file:`prj_release.conf` file for the release version.`

Reply:
By adding a `CONFIG_TRANSPORT_BIS=y` to the prj.conf file we actually modify a value of a Kconfig option.